### DISCUSSION TITLE
The connection is not allowed, The remote device is disabled. errror custom client connection

### DISCUSSION THREAD

**Discussion by mesuttavukcu** [rustdesk/rustdesk-server-pro#430] [General]
Yeni yüklemelerde The connection is not allowed, The remote device is disabled. hatası alıyorum.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/182

**Reply by mesuttavukcu** [reply]
I purchased a 10-user RustDesk license with the expectation that each of my users could support our clients. I have over 500 clients, and the process involves clients providing their ID so we can connect and resolve issues. I set up my own server under these terms and now face restrictions that prevent us from connecting to unmanaged clients. Please address this urgently, as this limitation severely impacts our support capabilities.
